packages/react/src/runtimes/local/LocalThreadRuntimeCore.tsx

```
@@ -159,7 +171,13 @@
       this._suggestionsController = null;
 
       do {
-        message = await this.performRoundtrip(parentId, message, runConfig);
+        message = await this.performRoundtrip(
+          parentId,
+          message,
+          runConfig,
+          runCallback,
+        );
+        runCallback = undefined;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)